scripts/integration_test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Runs integration tests on a given runtime image
+
+dir=`dirname $0`
+projectRoot=$dir/..
+testAppDir=$projectRoot/test-application
+deployDir=$testAppDir/target/deploy
+
+imageUnderTest=$1
+if [ -z "${imageUnderTest}" ]; then
+  echo "Usage: ${0} <image_under_test>"
+  exit 1
+fi
+
+# build the test app
+pushd $testAppDir
+mvn clean install
+popd
+
+# deploy to app engine
+pushd $deployDir
+export STAGING_IMAGE=$imageUnderTest
+envsubst < Dockerfile.in > Dockerfile
```

Comment:
I'm actually not crazy about that idea - is the motivation that you want to make this work across-platform? That wasn't really one of my goals with this PR, and if that was something we wanted to do, I'd probably want to rethink this bash-heavy approach to running these tests. 

Also, at some point in the near future, the integration test framework will be able to deploy the app being tested, and so we would need to go back and undo this change. (because the framework would need to do the templating of the `${STAGING_IMAGE}` variable).